targets/targets.json

```diff
@@ -374,8 +374,13 @@
             "SERIAL",
             "SLEEP",
             "SPI",
-            "SPISLAVE"
+            "SPISLAVE",
+            "USTICKER"
         ],
+        "overrides": {
```

Comment:
Tickless doesn't work without RTOS...

Reply:
@bulislaw 

I needed to add `tickless-from-us-ticker` to avoid runtime assertion here:

https://github.com/ARMmbed/mbed-os/blob/6b89d16dd374a3669b892da8e9dd485f601dbaa6/platform/source/mbed_os_timer.cpp#L55

Any suggestion to avoid this assertion for the non-RTOS target?